### PR TITLE
Check skip support

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- [x] fix help messages in tests to make CentosCI happy:

### CODE REVIEW

File: colin/core/colin.py [hunk 1]
```diff
@@ -23,12 +23,23 @@
...
+    target_type,
+    tags=None,
+    ruleset_name=None,
+    ruleset_file=None,
+    ruleset=None,
+    logging_level=logging.WARNING,
+    checks_paths=None,
+    pull=None,
+    insecure=False,
+    skips=None,
```
Comment (jpopelka) [inline colin/core/colin.py]: probably not an issue, but the comma after the last argument (not only here) looks strange to me

File: colin/cli/colin.py [hunk 2]
```diff
@@ -59,8 +59,10 @@ def cli():
               help="Enable debugging mode (debugging logs, full tracebacks).")
 @click.option('--json', type=click.File(mode='w'),
               help="File to save the output as json to.")
-@click.option('--stat', '-s', is_flag=True,
+@click.option('--stat', is_flag=True,
               help="Print statistics instead of full results.")
+@click.option('--skip', '-s', multiple=True, type=click.STRING,
+              help="Name of the check to skip.")
```
Comment (TomasTomecek) [inline colin/cli/colin.py]: Unfortunately it's not clear from `--help` that the option can be repeated:
```
  -s, --skip TEXT              Name of the check to skip.
```

File: colin/core/colin.py (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/core/colin.py]: I've tried the [black](https://github.com/ambv/black) for the reformatting. I can remove them if you wish..;-)

File: colin/cli/colin.py (hunk 2 above)
Comment (TomasTomecek) [inline colin/cli/colin.py]:
```suggestion
              help="Name of the check to skip. (this option is repeatable)")
```

File: colin/cli/colin.py [hunk 3]
```diff
@@ -141,6 +144,8 @@ def check(target, ruleset, ruleset_file, debug, json, stat, tag, verbose,
               help="Enable debugging mode (debugging logs, full tracebacks).")
 @click.option('--json', type=click.File(mode='w'),
               help="File to save the output as json to.")
+@click.option('-s', '--skip', multiple=True, type=click.STRING,
+              help="Name of the check to skip.")
```
Comment (TomasTomecek) [inline colin/cli/colin.py]:
```suggestion
              help="Name of the check to skip. (this option is repeatable)")
```

File: colin/core/colin.py (hunk 1 above)
Comment (TomasTomecek) [reply colin/core/colin.py]: I actually tend to this on my own as well: so I'm personally fine with it -- it's especially very useful when you keep adding more and you forget to add comma

**Comment by lachmanfrantisek** [comment]
@TomasTomecek Is it OK now?

**Comment by TomasTomecek** [comment]
CI is happy, I am happy, Jirka seems to be happy, everyone is happy. Happy!

**Comment by lachmanfrantisek** [comment]
@jpopelka @TomasTomecek Thanks for the review!

**Pull request merged**